packages/cozy-client/src/CozyClient.js

```diff
@@ -1761,13 +1761,16 @@ instantiation of the client.`
    * @returns {Promise<void>}
    */
   async loadInstanceOptionsFromStack() {
-    const { data } = await this.query(
-      Q('io.cozy.settings').getById('io.cozy.settings.capabilities')
-    )
+    const results = await Promise.all([
+      this.query(
+        Q('io.cozy.settings').getById('io.cozy.settings.capabilities')
+      ),
+      this.query(Q('io.cozy.settings').getById('io.cozy.settings.instance'))
+    ])
```

Comment:
Eventually, we might also implement a `getAll` in the settings collection, but let's keep it like this for now